docs/eessi.md

```diff
@@ -0,0 +1,92 @@
+# EESSI Guide
+
+## How to Load EESSI
+EESSI can be initialised using either of the following methods:
+
+1. Load the EESSI Environment Module.
+
+    The following command configures Lmod for your system and automatically loads the EESSI module:
+    ```[bash]
+    source /cvmfs/software.eessi.io/versions/2023.06/init/lmod/bash
+    ```
+    If Lmod is already installed, the following steps can be used to set up EESSI.
```

Comment:
`lmod` is always installed in slurm appliances, so we don't need the "if" and "check" bits.